tests/unittests/test_util.py

```diff
@@ -956,6 +956,12 @@ def test_get_proc_ppid(self):
         self.assertEqual(my_ppid, util.get_proc_ppid(my_pid))
 
 
+@mock.patch('os.uname')
+def test_kernel_version(m_uname):
+    m_uname().release = '5.6.19-300.fc32.x86_64'
+    assert util.kernel_version() == (5, 6)
```

Comment:
This test looks good, but I'd like to see us test multiple different inputs/expected outputs. pytest allows us to do this with test parameterisation; rewriting the current test to use it would look something like this:

```suggestion
@pytest.mark.parametrize("uname_release,expected", [
    ('5.6.19-300.fc32.x86_64', (5, 6)),
])
def test_kernel_version(m_uname, uname_release, expected):
    m_uname.return_value.release = uname_release
    assert expected == util.kernel_version()
```

An Ubuntu kernel version would be worth testing: `4.15.0-101-generic`, and I have another VPS with a differently formatted kernel version: `3.10.0-1062.12.1.vz7.131.10`.